src/wide/f32x8_t.rs

```diff
@@ -366,6 +500,11 @@ impl f32x8 {
                     f32x4_div(one, f32x4_sqrt(self.0)),
                     f32x4_div(one, f32x4_sqrt(self.1)),
                 )
+            } else if #[cfg(all(feature = "simd", target_arch = "aarch64", target_feature = "neon"))] {
+                // TODO: Is an estimate fine?
+                unsafe {
+                    Self(vrsqrteq_f32(self.0), vrsqrteq_f32(self.1))
```

Comment:
For recip and recip_sqrt, there are neon instructions that provide an estimate. I don't know if it's good enough for our purposes.